causing/model.py

```diff
@@ -50,15 +53,17 @@ def __post_init__(self):
         self.qydim = utils.count_nonzero(self.idy)
         self.qdim = self.qxdim + self.qydim
 
-    def compute(self, xdat: np.array) -> np.array:
+    def compute(self, xdat: np.array, bias: float = 0, bias_ind: int = 0) -> np.array:
         """Compute y values for given x values
 
         xdat: m rows, tau columns
         returns: n rows, tau columns
         """
+        assert isinstance(bias, (float, int)), f"bias must be scalar, not {bias!r}"
         assert xdat.ndim == 2, f"xdat must be m*tau (is {xdat.ndim}-dimensional)"
```

Comment:
Ndim should be 2? Why?

Reply:
We want it to work across multiple xvals (e.g. companies) at the same time, so we need exactly one dimension in addition to the dimension for the xvars. If you only have a single xval, you can still put that into a matrix with the right shape.

The old code did a "smart" reshape, which made some wrong inputs work correctly, but caused wrong results without immediate errors in other cases. Debugging that was an unnecessary time sink. The solution is to be strict about the input.

Reply:
Fine, of course. I mixed it up with our "ndim"  variable..